tools/validators/ontology_validator/yamlformat/validator/unit_lib.py

```diff
@@ -220,28 +222,28 @@ class Unit(findings_lib.Findings):
     name: the full name (without namespace) of this unit
     measurement_type: the unit measurement type
     is_standard: whether this is the standard unit for the measurement type
-    context: the config file context for where this unit was defined
+    file_context: the config file context for where this unit was defined
   """
 
-  def __init__(self, name, measurement_type, is_standard=False, context=None):
+  def __init__(self, name, measurement_type, is_standard=False, file_context=None):
     """Init.
 
     Args:
       name: required string name for the unit
       measurement_type: required string indicating the unit measurement type
       is_standard: whether this is the standard unit for the measurement type
-      context: optional object with the config file location of this unit.
+      file_context: optional object with the config file location of this unit.
```

Comment:
for the init you changed please add docstring as in here

Reply:
I think the docstring is in the right place. It's below def and above super().